runtime.go

```diff
@@ -1421,6 +1425,12 @@ func (r *Runtime) SetRandSource(source RandSource) {
 	r.rand = source
 }
 
+// SetCurrentTimeGetter sets the current time getter function for this Runtime.
+// If not called, the default time.Now() is used.
+func (r *Runtime) SetCurrentTimeGetter(now Now) {
```

Comment:
Could you please rename it to SetTimeSource()?

Reply:
Done, I just have to figure out how to update the PR.